Phonebook.Frontend/src/app/shared/dialogs/user-information/user-information.component.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { UserInformationComponent } from './user-information.component';
+
+xdescribe('UserInformationComponent', () => {
```

Comment:
Up to write a test? If not not but would be cool :)

Reply:
all tests handling MatDialogs are disabled